### PR TITLE
Update Dart docs to alpha.33

### PR DESCRIPTION
**Pull request by kwalrath**
How does this look for updating the Dart docs to alpha.33?

The last guide didn't require an update because it doesn't show pubspec.yaml or main.dart.

### CODE REVIEW

File: public/docs/dart/latest/quickstart.jade [hunk 1]
```diff
@@ -286,6 +287,25 @@ p.
...
+      When you import <code>bootstrap.dart</code>,
+      you also get <code>dart:mirrors</code>,
+      a reflection library that
+      causes performance problems when compiled to JavaScript.
+      Don't worry,
+      the Angular transformer converts your entry points
+      so that they don't use mirrors.
+      The transformer doesn't convert other libraries in your app,
+      so be sure to
+      import <code>angular.dart</code> instead of <code>bootstrap.dart</code>
```
Comment (yjbanov) [inline public/docs/dart/latest/quickstart.jade]: angular2.dart

File: public/docs/dart/latest/quickstart.jade [hunk 2]
```diff
@@ -73,17 +73,31 @@ p.
...
 .l-main-section
 
+  .l-sub-section
+    h3 Top-level Angular 2 libraries
+
+    p.
+      The main Dart file for any Angular 2 app must import
+      <code>'package:angular2/bootstrap.dart'</code>.
+      If you put part of your app into one or more additional libraries,
+      those additional libraries must import <code>angular.dart</code>
```
Comment (yjbanov) [inline public/docs/dart/latest/quickstart.jade]: angular2.dart

**Comment by yjbanov** [comment]
A couple of typos. LGTM otherwise.

**Comment by kwalrath** [comment]
I'll fix these now and submit. Thanks for the review!

**Pull request merged**